packages/harmony/src/assets/styles/spacing.css

```diff
@@ -13,16 +13,28 @@ root {
   --unit-2: 8px;
   --unit-3: 12px;
   --unit-4: 16px;
-  --unit-5: 24px;
-  --unit-6: 32px;
-  --unit-7: 40px;
-  --unit-8: 48px;
-  --unit-9: 56px;
-  --unit-10: 64px;
-  --unit-11: 72px;
-  --unit-12: 80px;
-  --unit-13: 96px;
-  --unit-14: 112px;
-  --unit-15: 144px;
-  --unit-16: 160px;
+  --unit-5: 20px;
+  --unit-6: 24px;
+  --unit-7: 28px;
+  --unit-8: 32px;
+  --unit-9: 36px;
+  --unit-10: 40px;
+  --unit-11: 44px;
+  --unit-12: 48px;
+  --unit-13: 52px;
+  --unit-14: 56px;
+  --unit-15: 60px;
+  --unit-16: 64px;
+  --unit-17: 68px;
+  --unit-18: 72px;
+  --unit-19: 76px;
+  --unit-20: 80px;
+  --unit-21: 84px;
+  --unit-22: 88px;
+  --unit-23: 92px;
+  --unit-24: 96px;
+  
+  --unit-half: 2px;
+  --negative-unit: -4px;
+  --negative-unit-2: -8px;
```

Comment:
these will need the harmony prefix

Reply:
yep working on it rn. You can wait to review if you want